tests/tracer/telemetry/test_telemetry.py

```diff
@@ -1,37 +1,58 @@
 import os
 
+import httpretty
+import pytest
+
 from ddtrace.internal import telemetry
 from ddtrace.internal.service import ServiceStatus
 
 
-def test_enable():
-    telemetry.telemetry_writer.enable()
-    assert telemetry.telemetry_writer.status == ServiceStatus.RUNNING
+@pytest.fixture
+def telemetry_writer():
+    yield telemetry.telemetry_writer
+
+
+@pytest.fixture(autouse=True)
+def mock_send_request(telemetry_writer):
```

Comment:
Enabling/Disabling the telemetry writer queues and sends requests to the agent. We should mock the telemetry endpoint instead of sending events to the agent (the response does not matter, the response is validated in other tests).